Front-End/challenge2/script.js

```diff
@@ -5,3 +5,98 @@ TO-DO:
 - The calculator should be completely functional
 
 */
+
+window.onload = () => {
+  //Display Listener
+
+  let displayField = document.getElementById("display");
+  let operator = "";
+  let num1 = 0;
+	let settedNum1 = false;
```

Comment:
Format your code using a tool like prettier

Reply:
Use plain English when naming your variables. Instead of num1 and num2, use firstNumber and secondNumber